docs/UsersGuide/source/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -303,7 +303,9 @@ For example, on Hera, the command would be:
 
    The user must have an Intel compiler and MPI on their system because the container uses an Intel compiler and MPI. Intel compilers are now available for free as part of `Intel's oneAPI Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/hpc-toolkit-download.html>`__.
 
-After this command runs, the working directory should contain ``srw.sh`` and a ``ufs-srweather-app`` directory. 
+After this command runs, the working directory should contain ``srw.sh``, a ``ufs-srweather-app`` directory, and an ``ush`` directory.
+
+.. COMMENT: Check that the above is true for the dev containers...
```

Comment:
@EdwardSnyder-NOAA Can you speak to this?

Reply:
This change is needed here as well. The workflow_tools env expects the `ush` directory to be the same level as the `modulefiles` directory.